piccolo/testing/model_builder.py

```diff
@@ -159,29 +151,110 @@ def _randomize_attribute(cls, column: Column) -> t.Any:
             Column class to randomize.
 
         """
-        random_value: t.Any
-        if column.value_type == Decimal:
-            precision, scale = column._meta.params["digits"] or (4, 2)
-            random_value = RandomBuilder.next_float(
-                maximum=10 ** (precision - scale), scale=scale
-            )
-        elif column.value_type == datetime.datetime:
-            tz_aware = getattr(column, "tz_aware", False)
-            random_value = RandomBuilder.next_datetime(tz_aware=tz_aware)
-        elif column.value_type == list:
-            length = RandomBuilder.next_int(maximum=10)
-            base_type = t.cast(Array, column).base_column.value_type
-            random_value = [
-                cls.__DEFAULT_MAPPER[base_type]() for _ in range(length)
-            ]
-        elif column._meta.choices:
+        reg = cls.get_registry(column)
```

Comment:
This can probably go into the `else` block, as we don't use it if the column has choices.

Reply:
That's a valid observation. It seems that the presence of `reg` is a result of the requirements in the previous version, where it was needed for multiple `elif-else` blocks.

Reply:
I'm not sure if this is a good idea.
```python    
    @classmethod
    def _randomize_attribute(cls, column: Column) -> t.Any:
        reg = cls.get_registry(column)
        random_value = reg.get(enum.Enum, reg[column.value_type])()
        if isinstance(column, (JSON, JSONB)):
            return json.dumps({"value": random_value})
        return random_value


    @classmethod
    def _get_local_mapper(cls, column: Column) -> t.Dict[t.Type, t.Callable]:
	...
        if _choices := column._meta.choices:
            local_mapper[enum.Enum] = partial(
                RandomBuilder.next_enum, _choices)

        return local_mapper
```